src/app/features/checkout/components/payment/payment.component.html

```diff
@@ -2,19 +2,17 @@
   <div class="headline">Card Details</div>
 
   <ile-credit-card-simulator
-    class="credit-card"
+    class="credit-card card-right"
```

Comment:
Should it be on the left, right or center?
What looks better?

Reply:
You are the one with the better taste between us..